js/index.js

```diff
@@ -7,51 +7,51 @@ document.addEventListener('DOMContentLoaded', () => {
     const messageList = messageSection.querySelector('ul')
     const skillsSection = document.getElementById('skills');
     const skillsList = skillsSection.querySelector('ul');
-
-   
-    //add a label to the users message if he checked out 'friend' of 'emploeyr' radio button
     
+    //add a label to the users message if he checked out 'friend' of 'emploeyr' radio button
+
     const arrImg = []
     arrImg[0] = new Image
     arrImg[1] = new Image
     arrImg[0].src = "img/star_logo.png"
     arrImg[1].src = "img/heart_logo.jpg"
     arrImg[0].width = 16
     arrImg[1].width = 16
-    
+
+    //check if radio button is checked or not
     function checkRadioButton() {
         let radio = document.getElementsByName('radio')
```

Comment:
just use const if not reassigning. Doesn't matter that much, just better practice.